packages/in_app_purchase/in_app_purchase_ios/pubspec.yaml

```diff
@@ -0,0 +1,34 @@
+name: in_app_purchase_ios
+description: An implementation for the iOS platform of the Flutter `in_app_purchase` plugin. This uses the iOS StoreKit Framework.
+repository: https://github.com/flutter/plugins/tree/master/packages/in_app_purchase/in_app_purchase_ios
+version: 1.0.0
```

Comment:
let's release this as 0.1.0 as the first version.

Reply:
Good point. I make the change and also updated the CHANGELOG.md to reflect the same version number.